Doc/library/stdtypes.rst

```diff
@@ -1653,6 +1653,15 @@ expression support in the :mod:`re` module).
    from the "Alphabetic" property defined in the Unicode Standard.
 
 
+.. method:: str.isascii()
+
+   Return true if all characters in the string are ASCII, false otherwise.
+   ASCII characters have code points in the range U+0000-U+007F.
```

Comment:
Maybe you can simplify the description as:

Return true if all characters have code points in the range U+0000-U+007F or if the string is empty, false otherwise.